test/differential/LogCompression.t.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+
+import { LogCompression } from "src/libraries/LogCompression.sol";
+
+contract LogCompressionTest is Test {
+
+    address private _balancerLogCompression = address(200);
+
+    function setUp() public
+    {
+        // we use getDeployedCode since the library contract is stateless
+        bytes memory lBytecode = vm.getDeployedCode("./reference/balancer-v2-monorepo/pkg/solidity-utils/artifacts/contracts/helpers/LogCompressionWrapper.sol/LogCompressionWrapper.json");
+        vm.etch(_balancerLogCompression, lBytecode);
+    }
+
+    function testToLowResLog(uint256 aValue) external
+    {
+        // act
+        (bool lBool, bytes memory lRes) = _balancerLogCompression.staticcall(abi.encodeWithSignature("toLowResLog(uint256)", aValue));
```

Comment:
`lBool` is an example of hungarian variable names. Linus says these are only to be used by the brain damaged. As we are both smart cookies: `lBool` -> `lSuccess`

Reply:
lol okay yeah forgot about that again. Fixed